tests/js-modules/modules.py

```diff
@@ -268,11 +268,14 @@ def test_npm_app(network, args):
         )
         label = "label42"
         r = c.post(
-            "/app/wrapKeyRsaOaep",
+            "/app/wrapKey",
             {
                 "key": b64encode(aes_key_to_wrap).decode(),
-                "wrappingKey": wrapping_key_pub_pem,
-                "label": label,
+                "wrappingKey": b64encode(bytes(wrapping_key_pub_pem, "ascii")).decode(),
```

Comment:
Why do you b64 encode the wrapping key? PEM should be fine as a string.

Reply:
Nevermind, makes sense, to keep it uniform with AES.

Reply:
It would be very unusual to put a symmetric (AES) key into a PEM and I suspect most libraries would simply reject it because they don't handle `----- BEGIN AES `...

Reply:
Clarification: that's for the AES wrap and I just wanted to have the same format for all three wrapping functions, so I re-encode the PEM too. I guess we could try and look into the wrapAlgo and make it dependent on that.

Reply:
Makes sense, let's leave it as it is. It's just a test anyway.